src/preload/preload.ts

```diff
@@ -162,13 +160,13 @@ contextBridge.exposeInMainWorld("misc", miscApi);
 function request<C extends GetCommandIds<"user", "server", "request">>(
     ...args: GetCommandData<GetCommands<"user", "server", "request", C>> extends never ? [commandId: C] : [commandId: C, data: GetCommandData<GetCommands<"user", "server", "request", C>>]
 ): Promise<GetCommands<"server", "user", "response", C>> {
-    return ipcRenderer.invoke("tachyon:request", ...args);
+    return ipcRenderer.invoke("tachyon:request", ...args) as Promise<GetCommands<"server", "user", "response", C>>;
```

Comment:
These tachyon messages are the only remaining messages with casts. Unfortunately the trick we've used with conditional types to support 1 or 2 arguments gets lost when you pass it to the `Parameters` utility type, it's just a bit too magic.

 I couldn't get past it so just explicitly cast this one.